### PR TITLE
Quick Start Example and Gif

### PR DESCRIPTION
**Pull request by SteveLasker**

### CODE REVIEW

File: docs/quick-start.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+- Pull a reference artifact by embedding `oras discover`
+  ```shell
+  oras pull -a \
+      ${REGISTRY}/${REPO}@$( \
+        oras discover  \
+          -o json \
+          --artifact-type scan-result/example \
+          $IMAGE | jq -r .references[0].digest)
+  ```
+[oras-releases]:                    https://github.com/sajayantony/oras/releases
```
Comment (michaelb990) [inline docs/quick-start.md:62]: this should probably be in the `oras-project` org, right?

File: docs/quick-start.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  docker run -d -p ${PORT}:5000 ghcr.io/oras-project/registry:latest
+  ```
+- Build and Push `$IMAGE`
+  ```bash
+  docker build -t $IMAGE https://github.com/wabbit-networks/net-monitor.git#main
+  docker push $IMAGE
+  ```
+- Push an SBoM
+  ```bash
+  echo '{"version": "0.0.0.0", "artifact": "'${IMAGE}'", "contents": "good"}' > sbom.json
```
Comment (michaelb990) [inline docs/quick-start.md:24]: is this supposed to be the blob layer of the sbom?

Comment (SteveLasker) [reply docs/quick-start.md:24]: yup

File: docs/quick-start.md (hunk 1 above)
Comment (SteveLasker) [reply docs/quick-start.md:62]: _it should_, but we haven't gotten builds from main, just yet. This is under-way, but we didn't want to block on the spec release while the builds are being finalized.

**Review by michaelb990 — APPROVED** [review]

**Review by sajayantony — APPROVED** [review]

**Pull request merged**